### DISCUSSION TITLE
安卓14系统问题

### DISCUSSION THREAD

**Discussion by kimshift** [constanline/XQuickEnergy#532] [Q&A/问答时间]
前段时间更新安卓系统到14后，好像插件收能量时那个弹窗不怎么弹出来了，能量还正常收。你们有遇到过吗

**Comment by chengnan049** [comment]
指的是 Toast 吗？我这边是正常的，Android 14 AP1A 
不知道是我没设置好还是怎么的？我用本插件的时候，有一定概率不会收集能量（即使我手动打开森林）
